resource-aggregate/config.yaml

```diff
@@ -0,0 +1,80 @@
+log:
+  debug: false
+apis:
+  grpc:
+    address: "0.0.0.0:9100"
+    tls:
+      caPool: "/secrets/public/rootca.crt"
+      keyFile: "/secrets/private/cert.key"
+      certFile: "/secrets/public/cert.crt"
+      clientCertificateRequired: true
+    capabilities:
```

Comment:
what has snapshotTreshold and occ to do with grpc capabilities?

Reply:
I want to change it:
```
clients:
  aggregate:
     snapshotTreshold:
     occ:
     eventStore:
 ```
 what do you think ?